libs/nest/README.md

```diff
@@ -0,0 +1 @@
+# Nest Modules
```

Comment:
Is this README supposed to be just a heading?

Reply:
Yeah, that's the (lazy) current pattern for these middle folders, which are required for our gitbook scripts.